client/post-editor/editor-html-toolbar/index.jsx

```diff
@@ -214,21 +231,19 @@ export class EditorHtmlToolbar extends Component {
 		const content = tag.options && tag.options.paragraph
 			? '\n' + selfClosedTag + '\n\n'
 			: selfClosedTag;
-		this.updateEditorContent( before + inner + content + after );
+		this.updateEditorContent( before, inner + content, after );
 	}
 
 	insertHtmlTagWithText( tag ) {
 		const { before, after } = this.splitEditorContent();
-		this.updateEditorContent( before + this.openHtmlTag( tag ) + tag.options.text + this.closeHtmlTag( tag ) + after );
+		this.updateEditorContent( before, this.openHtmlTag( tag ) + tag.options.text + this.closeHtmlTag( tag ), after );
 	}
 
 	insertCustomContent( content, options = {} ) {
 		const { before, inner, after } = this.splitEditorContent();
 		this.updateEditorContent(
-			before + inner +
-			( options.paragraph ? '\n' : '' ) +
-			content +
-			( options.paragraph ? '\n\n' : '' ) +
+			before,
+			inner + ( options.paragraph ? '\n' : '' ) + content + ( options.paragraph ? '\n\n' : '' ),
 			after );
```

Comment:
I know you didn't add this, but we may want to put the closing parenthesis on the next line